Logging/composer.json

```diff
@@ -48,8 +49,5 @@
         "psr-4": {
             "Google\\Cloud\\Logging\\Tests\\": "tests"
         }
-    },
-    "conflict": {
-        "psr/log": ">=3"
```

Comment:
The diff doesn't show it, but the `config` block was duplicated.

Reply:
I see that now! Thanks for pointing that out